k8s/helm/yona-server/templates/01_configmap_properties.yaml

```diff
@@ -44,14 +45,15 @@ data:
     yona.ldap.accessUserDn={{ .Values.ldap.user_dn | default "cn=admin,dc=example,dc=local" }}
     yona.ldap.accessUserPassword={{ .Values.ldap.user_password | default "password" }}
     yona.sms.enabled= {{ .Values.sms.enabled | default "false" }}
-    yona.sms.defaultSenderNumber = {{ .Values.sms.default_sender_number | default "+17184169858" }}
+    yona.sms.defaultSenderNumber = {{ .Values.sms.default_sender_number | default "+12312312312" }}
     yona.sms.alphaSenderId = {{ .Values.sms.alpha_sender_id | default "Yona" }}
-    yona.sms.alphaSenderSupportingCountryCallingCodes = {{ .Values.sms.alpha_sender_supporting_country_calling_codes | default "+31 +49 +33" }}
+    yona.sms.alphaSenderSupportingCountryCallingCodes = {{ .Values.sms.alpha_sender_supporting_country_calling_codes | default "+31 +49 +33" | quote}}
```

Comment:
Ensure that the value is wrapped in quotes if needed.
Habbit - Needed for yaml generated, but not needed for raw application.properties generation - PR ammended to remove.